### PR TITLE
Accepted events for round 2, including some fixes in affiliations

### PR DESCRIPTION
**Pull request by eileen-kuehn**
This PR holds the currently accepted contributions from round 2 of call for abstracts.
Further, we now include the countries in the affiliations.

This PR is currently a draft, as we are still missing some decisions from round 2.

### CODE REVIEW

**Comment by catsmith** [comment]
Please do not approve this until I have contacted the people who have submitted by email. I think it is nicer for them to hear by email that by things being published on the website.

**Comment by Chilipp** [comment]
@catsmith, the submissions have been published on the website quite some while ago.

It's a bit sad that I just uploaded the PDFs to zenodo today (as this was discussed during the meeting last week). So the archived PDFs now have different affiliations than the website

**Comment by catsmith** [comment]
Yeah the older ones from the first round had already been published which is fine, they have also already been emailed. This pull request is for the latest round of submissions which we have not even finished reviewing, they have not been discussed in a meeting and none of them has been emailed.

**Comment by Chilipp** [comment]
Aah, sorry. My bad :sweat_smile:

File: _events/talks/event-015.md
Comment (Chilipp) [inline _events/talks/event-015.md:36]: these two links need to be formatted and linked in the text.

File: _events/talks/event-015.md
Comment (Chilipp) [inline _events/talks/event-015.md:25]: needs formatting

File: _events/talks/event-015.md
Comment (Chilipp) [inline _events/talks/event-015.md:38]:
```suggestion
```
this shouldn't be in the abstract, should it?

**Comment by eileen-kuehn** [comment]
@catsmith, this PR will stay in draft mode until everything is finished including your email. Only after this, we will start the publishing. We already followed this workflow during round 1. Hope this is ok, otherwise we can try to change the procedure for the next round.
Please note, that submitters can already see decisions in Indico as soon as conveners save those. However, they are not actively informed about it by the system.

**Comment by catsmith** [comment]
Everyone accepted in this round have now been emailed (including the workshops and panels) so this can be made live when you are ready.

File: _events/talks/event-015.md
Comment (eileen-kuehn) [reply _events/talks/event-015.md]: I included those, please check.

File: _events/talks/event-015.md
Comment (eileen-kuehn) [reply _events/talks/event-015.md]: I did some minor changes. Did I forget about something?

**Comment by catsmith** [comment]
Any chance we can take all timings of these pages on the website. We received an email querying why the panel was only assigned 15 minutes. Until these are accurate it would be better not to put anything at all.

**Comment by Chilipp** [comment]
> Any chance we can take all timings off these pages on the website

@catsmith what exactly do you mean? None of these events specify any timing at all

File: _events/discussions/event-013.md
Comment (Chilipp) [inline _events/discussions/event-013.md:10]:
```suggestion
    - name: Vanessa Sochat
```
from the YAML-syntax there cannot be duplicated assignments of the ``&speaker`` variable. It should only be the first one for technical reasons.

File: _events/discussions/event-013.md
Comment (Chilipp) [inline _events/discussions/event-013.md:15]:
```suggestion
    - name: Jurriaan Spaaks
```
see above.

**Comment by catsmith** [comment]
@Chilipp Sorry my mistake, it is the event on Indico not the website. The similar styling confused me. 

https://indico.scc.kit.edu/event/863/contributions/7413/

It would still be nice to take them off there while they are wrong or correct them. I will raised this with Eileen when she is back from her holiday maybe I can fix them as I accept them.

**Review by Chilipp — CHANGES_REQUESTED** [review]
the PDF generation currently fails (see https://github.com/SORSE/sorse.github.io/actions/runs/214009163) because of invalid YAML syntax. We need to change this @eileen-kuehn

File: _events/discussions/event-013.md
Comment (eileen-kuehn) [reply _events/discussions/event-013.md:15]: Will take care tomorrow, thanks for spotting.

**Comment by eileen-kuehn** [comment]
@catsmith, this is the default length in indico. Please go ahead and change as required. You have the rights to do so ;)
I will check tomorrow if there is any chance to remove the default length or otherwise just removing visibility.

**Comment by eileen-kuehn** [comment]
@catsmith, durations are not visible anymore. Hope this helps in reducing confusion of submitters :)

File: _events/discussions/event-013.md
Comment (Chilipp) [reply _events/discussions/event-013.md]: thanks a lot @eileen-kuehn! Thinking about it, I'd propose we make your life easier and just remove the `author: *speaker` and don't show the author profile at all in the upper left corner. The only advantage of keeping the _author_ is that the profile in the upper left corner can show other things, like website, github, etc.

Recap: The `author` is show in the sidebar 
image

and the `authors` (plural) are shown below the title
image

Comment (Chilipp) [reply _events/discussions/event-013.md]: If you like, I can fix this within this PR and make a PR to sorse-data-filter to remove the `author` field

Comment (eileen-kuehn) [reply _events/discussions/event-013.md]: I am currently sitting in a train, so happy to take care about it :) However, the `speaker` thing is easy to solve. So feel free to adapt the visualisation but for the moment I tend to leave the data as is to give us more flexibility.

Comment (Chilipp) [reply _events/discussions/event-013.md]: Alright :+1: I'll disable the author profile in the sidebar later when I am in front of the computer and we leave the `author` in the front matter :smiley:

**Review by Chilipp — APPROVED** [review]

File: _events/discussions/event-013.md
Comment (Chilipp) [inline _events/discussions/event-013.md:25]: @eileen-kuehn, I just realized: is [event-013](https://887-267395254-gh.circle-artifacts.com/0/SORSE/programme/discussions/event-013/index.html) a panel discussion? If so, then the correct team identifier is `panel`, not `discussions`, see https://github.com/SORSE/sorse.github.io/blob/master/_data/committee/programme_teams.yml
```suggestion
category: panels
```

**Review by Chilipp — CHANGES_REQUESTED** [review]
event-013 has the wrong category and should be moved to `_events/panels/event-013.md`, right?

File: _events/discussions/event-013.md
Comment (eileen-kuehn) [reply _events/discussions/event-013.md:25]: Hm...  that is even more complicated. It seems to be a *panel*, but was reviewed as a *discussion*.

Comment (eileen-kuehn) [reply _events/discussions/event-013.md:25]: I got it now. We have deviating options for contributions and tracks (tracks are needed for reviewing). The contribution types are more fine-grained than the tracks. I defined the tracks along the options on the website whereas the contribution types handle everything on its own. See image.

Screenshot 2020-08-20 at 08 14 35

So I think we should drop some of the contribution types? @cc-a, what do you think?
Internally I can just several contribution types to the same tracks to fix this concrete issue.

Comment (Chilipp) [reply _events/discussions/event-013.md]: Alternatively we can also add a _discussions_ to the website. This was just not part of the original programme teams

Comment (Chilipp) [reply _events/discussions/event-013.md]: I must have missed that we have a _Discussion session_ category. Who is part of that review team?

Comment (cc-a) [reply _events/discussions/event-013.md]: Yep they have a combined remit for both panels and discussions - https://docs.google.com/document/d/16OZLKW_SM0VvWpPHPnEbhbNbs6mzCvOdXB3zppsH1n4/edit

Comment (Chilipp) [reply _events/discussions/event-013.md]: then maybe it's the best to add a new programme team to the webpage. We'd just need a little text to go on https://sorse.github.io/programme/discussions/ (such as in https://sorse.github.io/programme/panels/)

Comment (Chilipp) [reply _events/discussions/event-013.md]: shall I ask on slack? We can revert ae10621 and merge this PR, even if the `discussions` page is still missing

Comment (cc-a) [reply _events/discussions/event-013.md]: So I don't think we want to drop any contribution types. Can the `discussions` page just be a redirect to the `panels` page as this covers both?

Comment (Chilipp) [reply _events/discussions/event-013.md]: hmm, why not just take the text from the `panels` page and put it on the `discussions` page. This would acknowledge the two different contribution types and it would be consistent with all the remaining teams.

Comment (cc-a) [reply _events/discussions/event-013.md]: Whichever is easiest I guess...

**Review by Chilipp — APPROVED** [review]

**Pull request merged**
